README.md

```diff
@@ -216,6 +220,8 @@ add_swagger_documentation \
    base_path: '/super/api'
 ```
 
+`host` and `base_path` are also accepting a `proc` or `lambda`
```

Comment:
Make it into a full sentence, like "The `host` and `base_path` options also accept ..".